src/test/java/org/fundacionjala/coding/richardseverich/movies/MoviesTest.java

```diff
@@ -0,0 +1,64 @@
+
+package org.fundacionjala.coding.richardseverich.movies;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by RichardSeverich on 5/31/2017.
+ */
+public class MoviesTest {
```

Comment:
Try to split your tests
Each class with implemented methods should have unit tests